ansible/roles/operator-pipeline/templates/openshift/tasks/generate-index.yml

```diff
@@ -63,6 +73,7 @@ spec:
         #! /usr/bin/env bash
         set -x
 
+        chmod 644 podman-inspect.json
```

Comment:
Why do we need to change the file permissions once more here?

Reply:
It's not strictly needed. It's just to avoid leaving world-writable files around.